Libraries/StyleSheet/StyleSheet.js

```diff
@@ -59,6 +60,8 @@ var StyleSheetValidation = require('StyleSheetValidation');
  * subsequent uses are going to refer an id (not implemented yet).
  */
 class StyleSheet {
+  static flatten:Function;
```

Comment:
space after colon likethe other type decls? also if you wanted to take this further you could maybe write: `static flatten: typeof flattenStyle` http://flowtype.org/docs/typeof.html